src/property-filter/__tests__/property-filter.test.tsx

```diff
@@ -406,9 +406,16 @@ describe('property filter parts', () => {
   });
 
   describe('count text', () => {
-    test('is hidden when there are no filtering tokens', () => {
+    test('is not displayed when count text is empty', () => {
+      const { propertyFilterWrapper: wrapper } = renderComponent({
+        countText: '',
+        query: { tokens: [{ propertyKey: 'string', value: 'first', operator: ':' }], operation: 'or' },
+      });
+      expect(wrapper.findResultsCount()).toBe(null);
+    });
+    test('is not displayed when there are no filtering tokens', () => {
       const { propertyFilterWrapper: wrapper } = renderComponent({ countText: '5 matches' });
-      expect(wrapper.findResultsCount()!.getElement()).toBeEmptyDOMElement();
+      expect(wrapper.findResultsCount()).toBe(null);
```

Comment:
This change can break customer tests as well. I will submit a dry-run before merging.

Reply:
LGTM, did you do the dry-run already?

Reply:
Not yet, but I will not merge unless the dry-run is green